### PR TITLE
nao_interfaces: 0.0.1-1 in 'galactic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by ijnek**
Increasing version of package(s) in repository `nao_interfaces` to `0.0.1-1`:

- upstream repository: https://github.com/ijnek/nao_interfaces.git
- release repository: https://github.com/ijnek/nao_interfaces-release.git
- distro file: `galactic/distribution.yaml`
- bloom version: `0.10.7`
- previous version for package: `null`

### CODE REVIEW

File: galactic/distribution.yaml [hunk 1]
```diff
@@ -1291,6 +1291,13 @@ repositories:
       url: https://github.com/MRPT/mrpt.git
       version: develop
     status: developed
+  nao_interfaces:
+    release:
```
Comment (jacobperron) [inline galactic/distribution.yaml:1295]: Consider adding a `source` entry.

Comment (ijnek) [reply galactic/distribution.yaml:1295]: I'm not too certain on how I could do this change with bloom without opening a new PR.
Would manually adding this to ``ijnek:bloom-nao_interfaces-0`` branch be appropriate?

```
    source:
      type: git
      url: https://github.com/ijnek/nao_interfaces.git
```

Comment (jacobperron) [reply galactic/distribution.yaml:1295]: You can manually push changes to this PR.

Comment (ijnek) [reply galactic/distribution.yaml:1299]: Great, done.

**Comment by jacobperron** [comment]
Merged with `master` to workaround https://github.com/ros/rosdistro/issues/29964

**Pull request merged**